lib/absinthe/phase/telemetry.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Absinthe.Phase.Telemetry do
+  @moduledoc false
+
+  use Absinthe.Phase
+
+  def run(blueprint, event: [:absinthe, :resolution, :start]) do
+    start_time = System.system_time(:millisecond)
+    start_time_mono = System.monotonic_time()
+    metadata = %{type: :absinthe_resolution_start}
```

Comment:
I'd love to see the query/mutation name and type in the metadata here. A function to get this information from the blueprint is below:
```
def get_operation_metadata(blueprint) do
    case Blueprint.current_operation(blueprint) do
        nil -> {name: "", type: ""}
        op -> {name: op.name, type: op.type}
     end
end
```